### PR TITLE
SouthPaw-(Celo Sage)-Crypto wallet the celo way

### PR DESCRIPTION
**Pull request by Southpaw0**

### CODE REVIEW

File: blog/02-02-23-Crypto-Wallets-The-Celo-Way/Crypto-Wallets-The-Celo-Way.md [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+
+## Requirements
+Follow these links to get the required knowledge you need to understand what this tutorial explains. 
+* https://docs.celo.org/general 
+* https://www.investopedia.com/terms/b/bitcoin-wallet.asp 
+* https://youtu.be/AQO7KePXUEQ
+
+
+
+# What are Crypto Wallets?
```
Comment (viral-sangani) [inline blog/02-02-23-Crypto-Wallets-The-Celo-Way/Crypto-Wallets-The-Celo-Way.md]: Please don't use sing `#` anywhere in your article.

**Review by viral-sangani — APPROVED** [review]

File: blog/31-01-23-Creating_Your_First_Smart_Contract_on_the_Celo_Blockchain_Creating_Your_First_Smart_Contract_on_the_Celo_Blockchain.md [hunk 2]
```diff
@@ -0,0 +1,279 @@
...
+      address[] public contributors; 
+}
+
+```
+
+In our contract, we have a variable `owner` of type `address`, which we will use to store the `address` of the contract creator/owner, we also created a `minimumContibution` variable of type `uint` that would receive the value for the minimum amount of contribution that can be made to the contract and also a list of `address` to store contributors who have donated to the smart contract.
+
+Next, we set up our contract’s constructor, which will require the value of the minimum amount a contributor can make to the smart contract, as its input.
+
+```Solidity
```
[the remark this answers is not in the document]
Comment (Ishanpathak1) [reply blog/31-01-23-Creating_Your_First_Smart_Contract_on_the_Celo_Blockchain_Creating_Your_First_Smart_Contract_on_the_Celo_Blockchain.md]: Got it, will make sure it doesn't happen in the future

File: blog/31-01-23-Creating_Your_First_Smart_Contract_on_the_Celo_Blockchain_Creating_Your_First_Smart_Contract_on_the_Celo_Blockchain.md [hunk 3]
```diff
@@ -0,0 +1,279 @@
...
+url: https://github.com/Southpaw0
+image_url:
+tags: [celosage, video]
+hide_table_of_contents: true
+slug: /tutorials/creating-your-first-smart-contract-with-celo
+---
+
+
+
+# Introduction​
```
[the remark this answers is not in the document]
Comment (Ishanpathak1) [reply blog/31-01-23-Creating_Your_First_Smart_Contract_on_the_Celo_Blockchain_Creating_Your_First_Smart_Contract_on_the_Celo_Blockchain.md]: I have already started working on this, It wont happen in the future

File: blog/02-02-23-Crypto-Wallets-The-Celo-Way/Crypto-Wallets-The-Celo-Way.md [hunk 4]
```diff
@@ -0,0 +1,178 @@
+---
+title: Crypto-Wallets-The-Celo-Way
+description: This tutorials teaches you all you need to know about Crypto Wallets 
+authors:
+  - name: Israel Okunaya
+url: https://github.com/Southpaw0
+image_url: ![crypto-wallets](https://user-images.githubusercontent.com/104994589/216370370-bb2fbf5d-bed1-48fb-b0fe-d12c2ccf2d33.png)
```
[the remark this answers is not in the document]
Comment (Ishanpathak1) [reply blog/02-02-23-Crypto-Wallets-The-Celo-Way/Crypto-Wallets-The-Celo-Way.md]: I was unaware of this thank you for bringing this in my notice

**Pull request merged**